user/uv/template.py

```diff
@@ -41,17 +41,14 @@ def init_build(self):
 
 
 def post_build(self):
-    for shell in ["bash", "fish", "nushell", "zsh"]:
-        with open(self.cwd / f"uv.{shell}", "w") as cf:
+    for cmd, shell in [
```

Comment:
nothing strictly wrong with this, but i think a nested loop would be a bit more readable

Reply:
fair